module/VuFind/src/VuFind/RecordDriver/SolrOverdrive.php

```diff
@@ -331,24 +333,34 @@ public function getOverdriveAvailability()
      */
     public function isCheckedOut()
     {
-        $this->debug(' ischeckout', [], true);
+        $this->debug('isCheckedOut', [], true);
         $overdriveID = $this->getOverdriveID();
         $result = $this->connector->getCheckouts(true);
         if ($result->status) {
             $checkedout = false;
             $checkouts = $result->data;
+            $result->data = [];
             foreach ($checkouts as $checkout) {
-                if (strtolower($checkout->reserveId) == $overdriveID) {
+                $this->debug("comparing: {$checkout->reserveId} to $overdriveID");
```

Comment:
This debug message seems potentially outdated, since you're doing something more nuanced now.